### ISSUE TITLE
A11y_Visual Studio Code Services_Docs Get Started_Theme_ScreenReader:Narrator is not announcing message which is visually present after submitting the feedback

### ISSUE THREAD

**Issue by divyabharadwaj92** [microsoft/vscode-docs#5103]
**GitHubTags**

#A11yMAS; #A11yTCS; #A11ySev2; #DesktopWeb; #Visual Studio Code Services; #BM-Visual Studio Code Services-Web-Jan2022; #Web; #Narrator; #WCAG1.3.1; #FTP; #STP; #ChromiumEdge;

**Environment Details**
**Application Name:** Visual Studio Code Services
**Windows Version:** Windows 11

**Repro Steps**

1. Launch the application Visual Studio Code Services
2. Navigate to "Docs" link.
3. From left navigation, select "Get Started".
4. Navigate to "Themes" link.
5. Tab till the feedback option 
6. Observed that Narrator is not announcing the status message which is visually present after submitting the feedback.(This issue observed in all the scenarios where the feedback option is available).

**Actual:**
The narrator is not announcing the message which is visually present after submitting the feedback.(This issue observed in all the scenarios where the feedback option is available).
NOTE: This issue is not reproducing in NVDA screenreader.

**Expected Result**
Narrator should announce the status message which is visually present after submitting the feedback.

**User Impact**
Screen reader users may not understand if the feedback is submitted or not.

**Narrator Attachment**
![Presentation1](https://user-images.githubusercontent.com/97668149/149907249-3192e14c-3884-4e65-989f-b7de881c89cd.gif)
.

**Comment by fsteffi** [comment]
fixed on https://code.visualstudio.com/docs/getstarted/themes#_next-steps
**GithubTags:**#Closed;

[5103-fixed.webm](https://user-images.githubusercontent.com/24580570/216560586-7b62318d-a8c3-4422-913a-7526a50d0f55.webm)
